roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -113,6 +113,40 @@
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
+  ansible.builtin.shell: "cat /var/lib/logrotate/logrotate.status"
+  register: logrotate_output
+  tags:
+    - skip_ansible_lint
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
+  ansible.builtin.find:
+    paths: /var/log
+    recurse: yes
+    patterns: "*.log"
+  register: log_files
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
+  ansible.builtin.set_fact:
+    unrotated_files: "{{ log_files.files | rejectattr('path', 'in', logrotate_output.stdout_lines) | map(attribute='path') | list }}"
```

Comment:
Does this rejectattr actually work?

Since `logrotate_output.stdout_lines` is a list of strings, it won't check for substring matches, but for equality. Unless I'm mistaken, this always returns the entire list of `log_files.files`.